packages/atclient/include/atclient/atkeys.h

```diff
@@ -82,7 +85,11 @@ int atclient_atkeys_set_encrypt_private_key_base64(atclient_atkeys *atkeys, cons
 int atclient_atkeys_set_self_encryption_key_base64(atclient_atkeys *atkeys, const char *selfencryptionkeybase64,
                                                    const size_t selfencryptionkeybase64len);
 
-int atclient_atkeys_set_enrollment_id(atclient_atkeys *atkeys, const char *enrollment_id, const size_t enrollment_id_len);
+int atclient_atkeys_set_apkam_symmetric_key_base64(atclient_atkeys *atkeys, const char *apkamsymmetrickeybase64,
+                                                   const size_t apkamsymmetrickeybase64len);
```

Comment:
Just a nit

`apkam_symmetric_key_base64` is the name of the struct, so the parameter name should be the same

Reply:
All the other signatures follow this naming convention though